config/config.go

```diff
@@ -104,11 +104,22 @@ type ServerConfig struct {
 	// development and debugging. The value needs to be the name of the container.
 	LogDockerTail string `ini:"db_log_docker_tail"`
 
+	// **DEPRECATED**: Please use ignore_schema_regexp instead, since that uses an
+	// optimized code path in the collector and can avoid long-running queries.
+	//
```

Comment:
We could consider outputting a warning message during collector test runs that tells customers to upgrade to the new setting - otherwise its not very discoverable that the previous setting is deprecated.

Reply:
Sure, I'll add that.